### PR TITLE
added Linq2db.identity

### PR DESCRIPTION
**Pull request by GonzoSpire**
Tracks list of tasks to do:
- [x] move project to main linq2db repository and obsolete it
- [x] update project to VS2019
- [x] update project to use C#8
- [x] update project to use NRT
- [ ] automate build/testing/deploy using azure (integrate into existing pipelines)
- [ ] address existing issues
- [ ] check if we need to update code from https://github.com/dotnet/aspnetcore/tree/master/src/Identity

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/linq2db/linq2db?pullRequest=2533) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/linq2db/linq2db?pullRequest=2533) before we can accept your contribution.<br/><br/>:x: GonzoSpire [sign now](https://cla.dotnetfoundation.org/linq2db/linq2db?pullRequest=2533)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/linq2db/linq2db?pullRequest=2533) it.</sub>

File: linq2db.sln.orig [hunk 1]
```diff
@@ -0,0 +1,623 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```
Comment (MaceWindu) [inline linq2db.sln.orig:2]: wut?

**Comment by MaceWindu** [comment]
#3886

**Pull request closed without merging**